scss/civicrm/contact/pages/_events.scss

```diff
@@ -26,6 +26,31 @@
     }
   }
 
+  .crm-button_qf_Search_refresh {
+    .crm-i {
+      top: 10px;
+    }
+
+    .crm-form-submit {
+      margin: 12.5px 0 0 !important;
+    }
+  }
+
+  .crm-event-search-form-block,
+  .crm-results-block {
+    border-radius: $border-radius-base;
```

Comment:
but the radius is applied to a generic header no? Isn't that header styled somewhere else already?